dvc/remote/s3.py

```diff
@@ -212,6 +212,15 @@ def exists(self, path_info):
         fname = next(self._list_paths(path_info, max_items=1), "")
         return path_info.path == fname or fname.startswith(dir_path.path)
 
+    def makedirs(self, path_info):
+        # We need to support creating empty directories, on S3, that means
+        # creating an object with an empty body and a trailing slash `/`.
+        #
```

Comment:
Would be nice to put that `s3api put-obj` command as an example to illustrate what we are talking about

Reply:
the `s3api put-object` is equivalent to `self.s3.put_object(Bucket=path_info.bucket, Key=dir_path.path, Body="")`, not sure if adding the `s3api` command would help.

Reply:
Yeah, good point. Ok, let's leave it as that.